student.rb

```diff
@@ -0,0 +1,17 @@
+module StudentModule
+  require_relative 'person'
+  include PersonModule
+
+  class Student < Person
+    attr_accessor :classroom
+
+    def initialize(classroom)
```

Comment:
- [ ] Also please make the definition take the necessary argument ( age, classroom, name = 'Uknown', parent_permision = true)
- [ ] also the super should take the (age, name, parent-permission = true)